src/openrct2/config/Config.h

```diff
@@ -176,7 +176,7 @@ struct NotificationConfiguration
     bool park_award;
     bool park_marketing_campaign_finished;
     bool park_warnings;
-    bool park_rating_warnings;
+    bool park_objective_warnings;
```

Comment:
make this change in a different pr.

Reply:
Ffs, if I remove all this, almost literally nothing is left, and whatever would be left would be different from what I had made that there's almost no point in reviewing it at all.